github/repos.go

```diff
@@ -305,10 +305,10 @@ func (s *RepositoriesService) Get(ctx context.Context, owner, repo string) (*Rep
 	return repository, resp, nil
 }
 
-// Get the contents of a repository's code of conduct.
+// GetCodeOfConduct gets the contents of a repository's code of conduct.
 //
 // GitHub API docs: https://developer.github.com/v3/codes_of_conduct/
```

Comment:
URL can point to the very endpoint:

https://developer.github.com/v3/codes_of_conduct/#get-the-contents-of-a-repositorys-code-of-conduct